src/Uno.UWP/Devices/Radios/Radio.Android.cs

```diff
@@ -0,0 +1,252 @@
+				if (netCaps is null)
+				{
+					return null;
+				}
+				if (netCaps.HasTransport(Android.Net.TransportType.Wifi))
+				{
+					oRadio.State = RadioState.On;
+				}
+				else
+				{
+					oRadio.State = RadioState.Off;
+				}
+
+				return oRadio;
+			}
+			else
+			{
+				// for Android API 1 to 28 (deprecated in 29)
+#pragma warning disable CS0618 // Type or member is obsolete
+				var netInfo = connManager.ActiveNetworkInfo;
+				if (netInfo is null)
+				{
+					oRadio.State = RadioState.Off;
+				}
+				else
+				{
+					if (netInfo.Type == Android.Net.ConnectivityType.Wifi ||
+						(netInfo.Type == Android.Net.ConnectivityType.Wimax))
+#pragma warning restore CS0618 // Type or member is obsolete
+					{
+						oRadio.State = RadioState.On;
+					}
+					else
+					{
+						oRadio.State = RadioState.Off;
+					}
+				}
+				return oRadio;
+			}
+
+
+		}
+
+		private static Radio GetRadiosMobile()
+		{
+			Android.Content.Context context = Android.App.Application.Context;
+			if (!context.PackageManager.HasSystemFeature(Android.Content.PM.PackageManager.FeatureTelephony))
+			{
+				return null;
+			}
+
+			var oRadio = new Radio();
+			oRadio.Kind = RadioKind.MobileBroadband;
+			oRadio.Name = "Cellular";  // I have "Cellular 6" (Lumia 532), but maybe "6" doesn't mean anything
+
+			if (!Windows.Extensions.PermissionsHelper.IsDeclaredInManifest(Android.Manifest.Permission.AccessNetworkState))
+			{
+				oRadio.State = RadioState.Unknown;
+				return oRadio;
+			}
+
+			var connManager = (Android.Net.ConnectivityManager)context.GetSystemService(Android.Content.Context.ConnectivityService);
+
+			if (Android.OS.Build.VERSION.SdkInt > Android.OS.BuildVersionCodes.LollipopMr1)
+			{
+				// available since API 23
+				var activeNetwork = connManager.ActiveNetwork;
+				if (activeNetwork is null)
+				{
+					return null;
+				}
+				var netCaps = connManager.GetNetworkCapabilities(activeNetwork);
+				if (netCaps is null)
+				{
+					return null;
+				}
+				if (netCaps.HasTransport(Android.Net.TransportType.Cellular))
+				{
+					oRadio.State = RadioState.On;
+				}
+				else
+				{
+					oRadio.State = RadioState.Off;
+				}
+
+				return oRadio;
+			}
+			else
+			{
+				// for Android API 1 to 28 (deprecated in 29)
+#pragma warning disable CS0618 // Type or member is obsolete
+				var netInfo = connManager.ActiveNetworkInfo;
+				if (netInfo is null)
+				{
+					oRadio.State = RadioState.Off;
+				}
+				else
+				{
+					if (netInfo.Type == Android.Net.ConnectivityType.Mobile)
+#pragma warning restore CS0618 // Type or member is obsolete
+					{
+						oRadio.State = RadioState.On;
+					}
+					else
+					{
+						oRadio.State = RadioState.Off;
+					}
+				}
+				return oRadio;
+			}
+		}
+
+
+		private async static Task<IReadOnlyList<Radio>> GetRadiosAsyncTask()
+		{
+			var oRadios = new List<Radio>();
+
+			var oRadio = GetRadiosBluetooth();
+			if (oRadio != null) oRadios.Add(oRadio); // yield oRadio;

```

Comment:
```suggestion
			if(GetRadiosBluetooth() is { } bluetoothRadio)
			{
			   oRadios.Add(radio);
			}
```

Reply:
GetRadiosBluetooth return Radio

Reply:
Testing for null works as well, it's just about avoiding the variable reuse.